tests/document/indexes.py

```diff
@@ -491,6 +509,9 @@ class BlogPost(Document):
 
         self.assertEqual(BlogPost.objects.count(), 10)
         self.assertEqual(BlogPost.objects.hint().count(), 10)
+        # here we seem to have find a bug in PyMongo 3.
```

Comment:
(GitHub comment in order to remove it from code)
Seems to be a PyMongo3 bug
The cursor first makes a SON out of the list of tuples
Then later reuses it and wonders why is it not a list of tuples